index.js

```diff
@@ -194,24 +195,24 @@ function purgePlayers(client) {
 }
 
 function convertToByte(num) {
-	num = num & 0xFF;
-	num = num > 127 ? num - 256 : num;
-	return num;
+	num &= 0xFF;
+	new_num = num > 127 ? num - 256 : num;
+	return new_num;
```

Comment:
```suggestion
	num = num & 0xFF;
	num = num > 127 ? num - 256 : num;
	return num;
```
don't need unrelated changes, that should be in a different PR.